x-pack/plugins/apm/server/lib/connections/get_connection_stats/get_destination_map.ts

```diff
@@ -48,21 +48,19 @@ type Destination = {
 // - for each span, find the transaction it creates
 // - if there is a transaction, match the dependency name (span.destination.service.resource) to a service
 export const getDestinationMap = ({
-  setup,
+  apmEventClient,
   start,
   end,
   filter,
   offset,
 }: {
-  setup: Setup;
+  apmEventClient: APMEventClient;
```

Comment:
This looks so much better. Now it's actually possible to guess what the method does from its arguments.